build/Helix/ProcessHelixFiles.ps1

```diff
@@ -3,11 +3,11 @@ Param(
     [string]$CollectionUri = $env:SYSTEM_COLLECTIONURI,
     [string]$TeamProject = $env:SYSTEM_TEAMPROJECT,
     [string]$BuildUri = $env:BUILD_BUILDURI,
-    [string]$OutPutFolder = "HelixOutput"
+    [string]$OutputFilePath = "HelixOutput"
```

Comment:
I would interpret "$OutputFilePath" to be a path to a file. But this is a path to a folder.

Reply:
This change was made since callers of this are using OutputFilePath not OutPutFolder.

Reply:
Ah, got it. I would make the reverse of this fix: update the call sites.